packages/editor/src/client/routes/articleList.tsx

```diff
@@ -23,16 +23,22 @@ import {
   FlexboxGrid,
   Input,
   InputGroup,
-  Icon,
   IconButton,
   Table,
   Modal,
   Button,
-  Message
+  Message,
+  Pagination
```

Comment:
https://rsuitejs.com/guide/v5-features/#removed-code-table-pagination-code-and-enhanced-code-pagination-code